apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -261,16 +267,23 @@
         }
       }
     },
-    "zone": {
+    "ZoneAssessmentJsonSchema": {
       "type": "object",
+      "additionalProperties": false,
```

Comment:
This and related additions of `additionalProperties: false` breaks ~1200 assessments in the wild with ~2000 total errors. We can't make this change as part of this PR.

Reply:
This has been resolved in the latest commits.